wagtail/admin/templates/wagtailadmin/notifications/task_state_approved.html

```diff
@@ -0,0 +1,8 @@
+{% extends 'wagtailadmin/notifications/base.html' %}
+{% load i18n %}
+
+{% block content %}
+    <p>{% blocktrans with title=page.get_admin_display_title|safe task=task.name %}The page "{{ title }}" has been approved in moderation stage "{{ task }}".{% endblocktrans %}</p>
```

Comment:
The HTML versions of the templates should not use `|safe` on the title, as we _do_ want `<` `>` `&` characters to be escaped.

Reply:
Thanks for catching that - fixed now.